wkcuber/api/Layer.py

```diff
@@ -60,6 +64,28 @@ def _assert_mag_does_not_exist_yet(self, mag):
                 )
             )
 
+    def set_bounding_box(
+        self, offset: Tuple[int, int, int], size: Tuple[int, int, int]
+    ):
+        self.set_bounding_box_offset(offset)
+        self.set_bounding_box_size(size)
+
+    def set_bounding_box_offset(self, offset: Tuple[int, int, int]):
+        size = self.dataset.properties.data_layers["color"].get_bounding_box_size()
+        self.dataset.properties._set_bounding_box_of_layer(
+            self.name, tuple(offset), tuple(size)
+        )
+        for _, mag in self.mags.items():
+            mag.view.global_offset = offset
```

Comment:
What is the expected behavior for views that were created before the boundingbox was changed?
I have now updated the size and offset of the views that are representing the different magnifications of the layer. However, if the user creates a view of a smaller boundingbox explicitly (with `ds.get_view(...)`)  (before the bbox was changed) then the "global boundingbox" of the view does not get updated. The reason is, that te layer itself does not keep track of all Views that have been created. If this is a problem, the View might need to look up the "global boundingbox" from the properties.json each time the `read` or `write` method is called.

Reply:
Good point! I think, it's alright if a view sticks to the global bbox which was present at the time when the view was created (that would also mean that voxel addresses stay consistent throughout the lifetime of a view, right?). Setting the bbox explicitly shouldn't be a frequent option. Also it should be done either before creating a view or after working with a view. In these cases, no harm can be done, anyway. So, it sounds fine to me :+1: